### ISSUE TITLE
bosh simulate does not represent the proper list options as it appears we cannot define a list where each option can only appear once.

### ISSUE THREAD

**Issue by shots47s** [boutiques/boutiques#337]
In representing something like this from ArgParse:

`parser.add_argument('--parcellations', help='Group2 option: cortical parcellation(s) to extract stats from.',
                    choices=["aparc", "aparc.a2009s"],
                    default=["aparc"],
                    nargs="+")`

In Boutiques is something like this:
    
      {
            "command-line-flag": "--parcellations",
            "default-value": [
                "aparc"
            ],
            "description": "Group2 option: cortical parcellation(s) to extract stats from.",
            "id": "parcellations",
            "list": true,
            "name": "parcellations",
            "optional": true,
            "type": "String",
            "value-choices": [
                "aparc",
                "aparc.a2009s"
            ],
            "value-key": "[PARCELLATIONS]"
        },

When I run bosh exec simulate, this can produce argument lists with repeating entries.  Recommend adding something that defines that the value-choices are unique.

You can test this with: 1484547 published on Zenodo.

**Comment by glatard** [comment]
Would a boolean specifying that a list must have unique elements solve that? In that case, in your example, the size of the list couldn't be larger than 2, correct?

**Comment by shots47s** [comment]
Yes, and yes.  However, at the moment, simulate may duplicate one or the other entry even in max and min are set to 2.  So the boolean is necessary for uniqueness.

**Comment by shots47s** [comment]
That being said, how many times do you give a list of choices for a list of variables, and have the same entry more than once?  If there is a way to make this the default but have a "non-unique" flag, that may be more appropriate.
